docs/topics/configmap-mission-intro.adoc

```diff
@@ -5,9 +5,14 @@ This mission shows you how to:
 * Use the configuration provided by the `ConfigMap` within an application.
 * Deploy changes to the `ConfigMap` configuration of running applications.
 
+.About Externalized Configuration
+It is important for an application's configuration to be externalized and separate from its code. This allows for the application's configuration to change as it moves through different environments while leaving the code unchanged. This also keeps sensitive or internal information out of your codebase and version control. Many languages and application servers provide environment variables to support externalizing an application's configuration.
```

Comment:
* Replace _an application's configuration_ with _the configuration of an application_ or _the application configuration_.
* I am not entirely sure if an application _moves_ through environments. I would rather say it is _deployed_ or _executed_ there.
* Replace _codebase_ with _code base_. [IBM Style Guide]